getting_started/administrators.adoc

```diff
@@ -33,7 +33,16 @@ up, you can continue by choosing one of the following installation methods.
 
 == Installation Methods
 
-Choose one of the following installation methods that works best for you.
+Before moving forward, you can choose the number of cores the OpenShift server should use:
++
+----
+$ export GOMAXPROCS=n
```

Comment:
They can choose this post-install. It also needs to be set in whatever environment the server will run in (so if they run in a docker container, it has to be set in the docker container, etc).